internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -532,7 +534,7 @@ func generateQueries(ctx context.Context, cluster *v1beta1.PostgresCluster) stri
 	// }
 
 	for _, queryType := range baseQueries {
-		queriesContents, err := os.ReadFile(fmt.Sprintf("%s/queries_%s.yml", pgmonitor.GetQueriesConfigDir(), queryType))
+		queriesContents, err := os.ReadFile(fmt.Sprintf("%s/queries_%s.yml", pgmonitor.GetQueriesConfigDir(log), queryType))
```

Comment:
:thought_balloon: you could grab this path once then just use a variable - instead of having to change each line every time something changes